src/App.tsx

```diff
@@ -1,14 +1,63 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import type { Todo } from './types/Todo';
+import { StatusFilter } from './types/StatusFilter';
 
 export const App: React.FC = () => {
```

Comment:
React.FC is not needed
https://github.com/typescript-cheatsheets/react?tab=readme-ov-file#section-2-getting-started